src/improvements/tasks/MultisigTask.sol

```diff
@@ -893,7 +921,9 @@ abstract contract MultisigTask is Test, Script, ITask {
     /// check the state changes applied by the task. This function can check
     /// that only the nonce changed in the parent multisig when executing a task
     /// by checking the slot and address where the slot changed.
-    function checkStateDiff(VmSafe.AccountAccess[] memory) internal view virtual {}
+    function checkStateDiff(VmSafe.AccountAccess[] memory) internal view virtual {
+        revert("checkStateDiff must be implemented");
```

Comment:
This tricks solc into think subsequent code is reachable since it does not statically analyze `false`. This will let you avoid the need to suppress warnings, which is safer
 `foundry.toml` to ensure
```suggestion
        require(false, "checkStateDiff must be implemented");
``1